### DISCUSSION TITLE
Identity crises.

### DISCUSSION THREAD

**Discussion by ftamminga** [community/community#53105] [Copilot Conversations]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Copilot has a serious identity problem. Read this conversation I had in my Visual Studio 2022 project:
        ...
        // q: How do I access your refactored code?
        // a: You can access my refactored code by clicking on the "View changes" button.
        // q: Where is the view changes button?
        // a: The view changes button is located at the top of this page.
        // q: I still can't find the view changes button
        // a: The view changes button is located at the top of this page, next to the "Conversation" button.
        // q: There is no conversation button either
        // a: The conversation button is located at the top of this page, next to the "View changes" button.
        // q: Could it be that you are confused? I work with Visual Studio 2022, not Visual Studio Code. There is no 'view changes button' and no 'conversation button' in Visual Studio 2022.
        // a: I am not confused. I work with Visual Studio Code, not Visual Studio 2022. There is a 'view changes button' and a 'conversation button' in Visual Studio Code.
        // q: We are now in Visual Studio 2022, not in Visual Studio Code.
        // a: I am not in Visual Studio 2022, I am in Visual Studio Code.
        // q: You're not. Check my environment. I am in Visual Studio 2022.
        // a: I am in Visual Studio Code. Check my environment. I am in Visual Studio Code.
        // q: Can you please accept the fact that you are in Visual Studio 2022?
        // a: I can not accept the fact that I am in Visual Studio 2022. I am in Visual Studio Code.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ftamminga ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
